### ISSUE TITLE
Can't change fontface

### ISSUE THREAD

**Issue by ks74** [nptr/msstyleEditor#15]
Hi again, thanks for the application, one more limitation: I want to change font for some system stuff like Start Menu, but wherever I see fonts in an msstyles file, I only have a limited list of other fonts to select from, they're all Segoe of various sizes and styles. But I can't change font face to Tahoma or smth else. I opened the msstyle file in ResHack and found a string table with all the Segoe fonts, then I replaced one of the items to Arial, just for test, and complied it and saved it. But this had no effect on the msstyle file.

**Comment by ks74** [comment]
Thank you, it worked. I could only change the font of the taskbar, control panel and the date/time area (string 502). Maybe some other things were affected too but I didn't notice yet. Alas, Start menu and many other applications do not depend on the styles, they use Calibri, not Segoe at all. 

I checked the dlls with ResHack, they don't have any fonts, by the way.

**Comment by nptr** [comment]
I saw the .msstyles and msstyles.mui to have one at least, so i suspected the shellstyle.dll can have them too, but maybe you are right.

**Comment by ks74** [comment]
You're right, what I checked was the dlls of the theme I use (AeroByDesign) and they don't have strings, while the standard msstyles do have the same list of fonts.

**Comment by ks74** [comment]
One more note; ResHack and your application have different string ids, they differ by 1. 
For example, in ResHack "Segoe UI, 12, Quality:ClearType" had id 503 and your app has it as
(Segoe UI, 12, Quality:ClearType (502))
Not sure whose fault it is.

**Comment by nptr** [comment]
ResHack is correct. As stated above i hardcoded the fontname <> ID translation, and in this case my table is simply wrong. I'll definitely have to fix this.

**Comment by nptr** [comment]
Since version 1.4.0.0 i am loading the string table and show the correct font name in the dropdown list.
That fixes the "bug" part of this issue.

There is still no way to edit the table yet because it might be located in a .mui file which i can indirectly read by opening the .msstyles file, but not write. Loading the .mui manually creates conceptional challenges which I don't want to tackle right now. So to add or change font entries, the use of a PE resource editor is still necessary

**Comment by ks74** [comment]
Thanks, great. I have completed my big work of elaborating Windows styles, yet the fix is good anyway.
